packages/pwa/app/components/forms/promo-code-fields.jsx

```diff
@@ -25,7 +25,7 @@ const PromoCodeFields = ({form, prefix = '', ...props}) => {
                     isLoading={form.formState.isSubmitting}
                     disabled={code?.length < 3}
                 >
-                    <FormattedMessage defaultMessage="Apply" />
+                    <FormattedMessage defaultMessage="Apply" id="promo_code_field.button.apply" />
```

Comment:
I think this is another pluralization issue. I might be wrong, but if I only new the id, I'd assume that the component there this id is being used in would be `components/forms/promo-code-field.jsx` but it's actually `components/forms/promo-code-fields.jsx`